src/math/fma.rs

```diff
@@ -218,7 +218,11 @@ mod tests {
             -0.00000000000000022204460492503126,
         );
 
-        assert_eq!(fma(-0.992, -0.992, -0.992), -0.007936000000000007,);
+        let result = fma(-0.992, -0.992, -0.992);
+        //force rounding to storage format on x87 to prevent superious errors.
+        #[cfg(all(target_arch = "x86", not(target_feature = "sse2")))]
+        force_eval!(result);
```

Comment:
I agree that would be better, but I don't think the current force_eval! macro supports that construction (I'm fairly sure I tried that first before settling on the current code).

Reply:
Ah you're right. You just need to make a small change to the macro definition (in src/math/mod.rs) to remove the `;` at the end and then it will work.

Reply:
Done